internal/tools/bigquery/bigquerygettableinfo/bigquerygettableinfo.go

```
@@ -137,6 +164,16 @@ func (t Tool) Invoke(ctx context.Context, params tools.ParamValues, accessToken
 		return nil, fmt.Errorf("invalid or missing '%s' parameter; expected a string", tableKey)
 	}
 
+	// Handle fully-qualified dataset ID in the dataset parameter.
```

Should we throw an error if this split is not as expected?